assets/src/design-system/components/input/index.js

```diff
@@ -39,13 +41,32 @@ const Hint = styled(Text)`
   margin-top: 12px;
 `;
 
+const SuffixContainer = styled.div`
```

Comment:
I think this one should extend `<Typography.Text size="small" />` to have correct text styling for letter suffixes.